docs/serving/using-a-custom-domain.md

```diff
@@ -1,29 +1,32 @@
-# Default domain name settings
+# Configuring domain names
 
-The fully qualified domain name for a route by default is `{route}.{namespace}.{default-domain}`. Knative Serving routes use `example.com` as the default domain.
+You can customize the domain of an individual Knative Service, or set a global default domain for all Services created on a cluster. The fully qualified domain name for a route by default is `{route}.{namespace}.example.com`.
 
-You can change the default domain by modifying the [`config-domain` ConfigMap](https://github.com/knative/serving/blob/main/config/core/configmaps/domain.yaml).
+## Configuring a domain for a single Knative Service
 
-!!! warning
-    Customizing a domain using this ConfigMap affects your cluster globally. If you want to customize the domain of an individual Service, use [`DomainMapping`](services/custom-domains.md) instead.
+If you want to customize the domain of an individual Service, see the documentation about [`DomainMapping`](services/custom-domains.md).
 
-## Procedure
+## Configuring the default domain for all Knative Services on a cluster
+
+You can change the default domain for all Knative Services on a cluster by modifying the [`config-domain` ConfigMap](https://github.com/knative/serving/blob/main/config/core/configmaps/domain.yaml).
+
+### Procedure
 
 1. Open the `config-domain` ConfigMap in your default text editor:
 
-     ```bash
-     kubectl edit configmap config-domain -n knative-serving
-     ```
+    ```bash
+    kubectl edit configmap config-domain -n knative-serving
+    ```
 
-1. Edit the file to replace `example.com` with the domain you want to use, then remove the `_example` key and save your changes. In this example, `mydomain.com` is configured as the domain for all routes:
+1. Edit the file to replace `example.com` with the domain you want to use, then remove the `_example` key and save your changes. In this example, `knative.dev` is configured as the domain for all routes:
```

Comment:
I think we should use a different domain than `knative.dev` for this page. why `mydomain.com` bad to use?

Reply:
It was the specific request of the issue https://github.com/knative/docs/issues/4352 - I thought it would be confusing to use changing from `example.com` to `example.org` since they're so similar, so went with `knative.dev` since this was the other option. It shouldn't matter since a user shouldn't be using any of the example values anyway, but we should make a decision here about what we're using, because we end up with back and forth every time I try to fix this issue 😂

Reply:
First thing that popped in my head was something like `customdomain.example.com`, but then adding a subdomain might not be any less confusing...  that said, all else being equal, I think it's probably better to use a domain we own in our docs as opposed to one we don't.